articles/azure/azs-how-deploy-sql-template-powershell.md

```diff
@@ -146,47 +152,47 @@ The images used to create this deployment are:
 
 3. Log in to your Azure Stack *Subscription* with Service Principal Name (SPN)
 
-4. Check if Resource Group exits and create one if it does not.
+4. Check if the resource group exists and create one if it does not.
 
-5. Test Deployment JSON
+5. Test deployment JSON.
```

Comment:
Can we elaborate on what this step actually means?

Reply:
Test the JSON ARM template to ensure it is a valid deployment.